Reply/app/src/main/java/com/materialstudies/reply/ui/MainActivity.kt

```diff
@@ -97,35 +90,28 @@ class MainActivity : AppCompatActivity(),
             }
         }
 
-        bottomNavDrawer.apply {
-            addOnSlideAction(HalfClockwiseRotateSlideAction(binding.bottomAppBarChevron))
-            addOnSlideAction(AlphaSlideAction(binding.bottomAppBarTitle, true))
-            addOnStateChangedAction(ShowHideFabStateAction(binding.fab))
-            addOnStateChangedAction(ChangeSettingsMenuStateAction { showSettings ->
-                // Toggle between the current destination's BAB menu and the menu which should
-                // be displayed when the BottomNavigationDrawer is open.
-                binding.bottomAppBar.replaceMenu(if (showSettings) {
-                    R.menu.bottom_app_bar_settings_menu
-                } else {
-                    getBottomAppBarMenuForDestination()
-                })
-            })
-
-            addOnSandwichSlideAction(HalfCounterClockwiseRotateSlideAction(binding.bottomAppBarChevron))
-            addNavigationListener(this@MainActivity)
-        }
-
-        // Set up the BottomAppBar menu
-        binding.bottomAppBar.apply {
-            setNavigationOnClickListener {
-                bottomNavDrawer.toggle()
+        val bottomNavigation: BottomNavigationView = findViewById(R.id.bottom_navigation)
```

Comment:
With databinding you don't need to find the views yourself. This can be deleted and then you can get the `bottomNavigation` view from the `binding` object. As in:

`binding.bottomNavigation.setOnItemSelectedListener { item ->`